PLIST_INSECURE_CONFIG/description.md

```diff
@@ -0,0 +1,2 @@
+When file sharing is enabled, the entire Documents folder is used for file sharing. Files that that are not intended for user access via the file sharing feature should be stored in another part of the application's bundle. An
```

Comment:
The title of the KB is not correct and this description has no mention of the feature enabled.
I also don't understand the threat here.

Reply:
for the title : PLIST_INSECURE_UI_FILE_SHARING ?